src/native/hash/bitfn.h

```diff
@@ -22,8 +22,7 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef BITFN_H
-#define BITFN_H
+#pragma once
```

Comment:
@infinity0: I believe the problem detailed above to be an obvious deficit in Solaris Studio C/C++, and believe we should labor to have this much-needed feature added to Solaris Studio C/C++, after which we can merge this PR, since I agree - with all of my heart - with your sentiment quoted below:
>> especially since the world at present has a lack of well-written security code
- and I would like to add that this is especially needed in the Solaris ecosystems of the world where software using `nocrypto` will most likely contribute extensively towards this goal by being the only secure code around.